### PR TITLE
Add features/docs for inject (4.17)

### PR DESCRIPTION
**Pull request by hMcLauchlan**
Added the optional probability of failure discussed at #1633, additionally updated some documentation to reflect patches getting into mainline.

### CODE REVIEW

**Comment by yonghong-song** [comment]
[buildbot, ok to test]

File: man/man8/inject.8 [hunk 1]
```diff
@@ -13,13 +13,6 @@ kernel. You should know what you're doing if you're using this tool.
 
 This makes use of a Linux 4.16 feature (bpf_override_return())
 
-Additionally, use of the kmalloc failure mode is only possible with 
-
-	commit f7174d08a5fc ("mm: make should_failslab always available for
-	fault injection")
-
-which is in mm-tree but not yet in mainline (as of 4.16-rc5).
```
Comment (yonghong-song) [inline man/man8/inject.8]: What is the status of this patch?

**Comment by 4ast** [comment]
LGTM

File: man/man8/inject.8 (hunk 1 above)
Comment (hMcLauchlan) [reply man/man8/inject.8]: It's in mainline.

**Pull request merged**